src/aws/output.tf

```diff
@@ -1,4 +1,4 @@
-output "cloudfront_url" {
+output "cloudfront_domain" {
```

Comment:
Where this is use ? Is it only display in the terminal when you run `plan` command ?

Reply:
Yes. Just for manual display. The name was incoherent with the `CLOUDFRONT_URL` setting in Django.